.github/workflows/lint_python.yml

```diff
@@ -51,5 +37,25 @@ jobs:
       - run: codespell . --ignore-words-list=eith,gae --skip=./.* --quiet-level=2
       - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
       - run: isort --recursive . || true
-      - run: pip install -r requirements.txt
-      - run: pytest .
+      - run: pip install -r requirements.txt -r test_requirements.txt
+
+      # Use the default MySQL server offered by Github Actions.
+      - run: mysql -uroot -proot -e "create user 'scott'@'localhost' identified by 'tiger'; create database webpy; grant all privileges on webpy.* to 'scott'@'localhost' with grant option;"
+
+      - run: |
+          createdb -h localhost -U postgres webpy
+          createuser -h localhost -U postgres -d scott
+          psql -h localhost -U postgres -d postgres -c "ALTER USER scott WITH ENCRYPTED PASSWORD 'tiger'"
+          psql -h localhost -U postgres -d postgres -c "ALTER DATABASE webpy OWNER TO scott"
+        env:
+          PGPASSWORD: postgres
+
+      # Run pytest and get detailed output for easy debugging if test failed.
+      # Env variables `WEBPY_DB_` are required for sql db connections.
+      - run: pytest -s -x -v .
```

Comment:
What are the cryptic options s, x, and v...  Do they have alternatives that are more self-explanatory?

Reply:
Fixed.